### ISSUE TITLE
Intermittent issue calling ML endpoint. 

### ISSUE THREAD

**Issue by mazer-rakham** [Azure-Samples/graphrag-legalcases-postgres#17]
I think I have narrowed it down to token usage, just trying to get a confirmation. I have deployed successfully to the cloud, can ask questions to the Vector Search only perfectly fine, Semantic Ranker works, but only if you give it time before hitting it again, the Graph Rag + Semantic Ranker I have not gotten to work, it fails every time. Is there a estimation token usage somewhere for me to reference in able to troubleshoot better?  
![image](https://github.com/user-attachments/assets/d3bad692-1120-4a4c-8e1e-be6c87d582cb)
![image](https://github.com/user-attachments/assets/71b1f144-7fa2-4b99-a30a-02e5c23c1e54)

The error from App Insights is a 404 when it is traced:
![image](https://github.com/user-attachments/assets/205d5030-6a9a-4d58-bbe4-a418a76982f2)

**Comment by serdarmicrosoft** [comment]
Thanks for reporting the issue @mazer-rakham . Has it been completely resolved after performing a hard refresh on your browser? While there is a 16K token limit for the Azure OpenAI model, I suspect your issue might be unrelated.

**Comment by mazer-rakham** [comment]
Yeah, if you hard refresh the browser it works on each of the 3 designs, so do vector only, hard refresh, semantic only, hard refresh, then combination, hard refresh and it works, if you try and do one, than the other, then it [object, Object] with the weird 404 ml endpoint in App Insights.  Also I had to do  alot of modification for it to deploy the postgres to the cloud and seed it correctly, I had to do a lot of py import changing and added a new setup script

**Comment by kavitha-19** [comment]
![Image](https://github.com/user-attachments/assets/457d1b5b-c06e-4eb1-9d11-2b3aa20a4827)


Even after hard refresh ,it responds only for the first question , from the next follow up questions I am getting Object Object error for semantic and graph RAG. Is there any solution to resolve this issue.

This the error from the container app log stream 

![Image](https://github.com/user-attachments/assets/82c3e2e3-5dad-4d36-be2e-774484382ece)
